log_outgoing_requests/formatters.py

```diff
@@ -1,30 +1,39 @@
 import logging
 import textwrap
 
+from django.conf import settings
+
 
 class HttpFormatter(logging.Formatter):
     def _formatHeaders(self, d):
         return "\n".join(f"{k}: {v}" for k, v in d.items())
 
+    def _formatBody(self, content: dict, request_or_response: str) -> str:
+        if settings.LOG_OUTGOING_REQUESTS_LOG_BODY_TO_STDOUT:
+            return f"\n{request_or_response} body:\n{content}"
+        return ""
+
     def formatMessage(self, record):
         result = super().formatMessage(record)
         if record.name == "requests":
             result += textwrap.dedent(
                 """
                 ---------------- request ----------------
                 {req.method} {req.url}
-                {reqhdrs}
+                {reqhdrs} {request_body}
 
                 ---------------- response ----------------
                 {res.status_code} {res.reason} {res.url}
-                {reshdrs}
+                {reshdrs} {response_body}
 
             """
             ).format(
                 req=record.req,
                 res=record.res,
                 reqhdrs=self._formatHeaders(record.req.headers),
                 reshdrs=self._formatHeaders(record.res.headers),
+                request_body=self._formatBody(record.req.body, "Request"),
+                response_body=self._formatBody(record.res.json(), "Response"),
```

Comment:
this will crash when the response body is not json-parseable, which will happen in OF as we make a lot of XML requests/responses.

I think you'll need `record.res.text()` or even the raw underlying bytes content

This should also work with a setting `LOG_OUTGOING_REQUESTS_LOGGABLE_CONTENT_TYPES`, with a default value of something along the lines of:

```py
LOG_OUTGOING_REQUESTS_LOGGABLE_CONTENT_TYPES = [
    "text/*",
    "application/json",
    "application/xml",
]
```
(exact mime types need to be looked up). You *don't* want to log binary content like file uploads.

This should be checked against the `request.headers["Content-Type"]` and `response.headers["Content-Type"]`. Additionally, you may want to add a setting for upper body size limits which you can get from the `Content-Length` HTTP header